extensions/wikibase/module/scripts/schema-alignment.js

```diff
@@ -1467,13 +1498,38 @@ SchemaAlignment._initField = function(inputContainer, mode, initialValue, change
   }
 
   var acceptDraggableColumn = function(column) {
+    var serviceUrl = null;
+  var service = null;
+  var serviceLogo=null;
+  if (reconConfig) {
+    serviceUrl =reconConfig.service;
+  }
+  if (serviceUrl) {
+    service = ReconciliationManager.getServiceFromUrl(serviceUrl);
+  }
+  if(service){
+    serviceLogo=service.logo;
+  }
```

Comment:
All those lines above seem a lot similar to the ones earlier in the file, so perhaps it would be worth introducing a new function to avoid this duplication? For instance, the function could take a recon service manifest object and return the URL of its logo, if a valid logo can be found in the service.